README.md

```diff
@@ -1,5 +1,4 @@
-![windows](https://github.com/imperialCHEPI/healthgps/actions/workflows/windows.yml/badge.svg)
-![Linux](https://github.com/imperialCHEPI/healthgps/actions/workflows/linux.yml/badge.svg)
+[![CI](https://github.com/imperialCHEPI/healthgps/actions/workflows/ci.yml/badge.svg)](https://github.com/imperialCHEPI/healthgps/actions/workflows/ci.yml)
```

Comment:
This might be the reason to keep it separated into two workflows, so it is explicit in which platform each thing runs. We know it is a purely cosmetic thing and that the tests are running in both, anyway, but I guess it might be relevant to consider it?

Reply:
Yeah, I was disappointed to find that after all this work there was no way to get separate badges Linux and Windows here :disappointed:. (It would be even nicer if we could get separate badges for the building and testing steps too.)

My feeling is that losing the separate badges is probably just the price we have to pay for this PR. It may be that in future we can figure out how to separate out the various CI steps and get the badges back. We should enforce tests passing before PRs can be merged anyway, so hopefully it should only ever be green in any case!

@jamesturner246 maybe we should have a chat about what settings we want for the repo in terms of conditions for PRs being merged etc.?

Reply:
Sure. At least testsuite gating, and maybe some valgrind and/or code linting. Happy to discuss ofc.